content/docs/hooks-reference.md

```diff
@@ -76,7 +76,7 @@ If your update function returns the exact same value as the current state, the s
 > Unlike the `setState` method found in class components, `useState` does not automatically merge update objects. You can replicate this behavior by combining the function updater form with object spread syntax:
 >
 > ```js
-> setState(prevState => {
+> useState(prevState => {
```

Comment:
Appreciate the feedback!

I think it makese more sense to include where `setState` is comming from. We don't want `useState` here but the updater function i.e.

```suggestion
> const [state, setState] = useState({});
> setState(prevState => {
```

Reply:
yes, of course! I confused myself)